Eat_and_Greet/src/comps/CircularButton/style.tsx

```diff
@@ -2,14 +2,26 @@ import { StyleSheet } from 'react-native';
 
 const CircularButtonStyles = StyleSheet.create({
     button: {
-        borderRadius: 50,
```

Comment:
Is there a use case for this button outside of this? and if so, make the border radius, width, height etc. variable.